### ISSUE TITLE
Loss

### ISSUE THREAD

**Issue by Anooyman** [smlaine2/tempens#4]
I run the program with one gpu, but find out that train loss, test loss and the acc do not change.  Even if I change the learning rate and the mini-batch size, they do not change either.
Is there some one meet the same problems?

**Comment by Anooyman** [comment]
Hi, these days I recheck the code and the output files. I found that while the number of Epoch raise, the training loss raise. And the training_err also raise too. Is there some mistakes about the code?
